lcmtypes/driver_status_t.lcm

```diff
@@ -0,0 +1,58 @@
+package robot_msgs;
+
+// message with metadata about robot driver
+
+struct driver_status_t {
+  // the timestamp in microseconds
+  int64_t utime;
+
+  // true if the driver is running and a connection 
+  // to the arm has been sucecssfully established
+  boolean driver_running;
+
+  // optional messagee describing error if driver is 
+  // not running
+  string err_msg;
+
+  // string describing robot's current mode
+  string robot_mode;
```

Comment:
uint/enum better? cc @aneksteind

Reply:
We have non-overlapping enums for the two drivers so I think string is OK for now, also much more readable in `lcm-echo` for the driver status